lib/Twig/Extensions/Grammar.php

```diff
@@ -8,6 +8,10 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
+/**
+ * @deprecated since version 1.5
```

Comment:
No, I don't want to flood the user with many notices. One in the main class is enough.